peepingtom/datablocks/simpleblocks/orientationblock.py

```diff
@@ -31,106 +31,38 @@ def _data_setter(self, data: np.ndarray):
             m = data.shape[-1]
             data = data.reshape((1, m, m))
 
-        return data
+        dims = ['x', 'y', 'z']
+        return xr.DataArray(data, dims=['n', 'spatial1', 'spatial2'],
+                            coords=(range(len(data)), dims[:data.shape[1]], dims[:data.shape[1]]))
```

Comment:
can you explain this coords thing? I had a quick look at the xarray docs and it's not immediately obvious to me what's going on here

Reply:
`coords` is the xarray equivalent of `index` in pandas. So while an image has:
```python
dims = ('z', 'y', 'x')
coords = None    # because there is no qualitative distinction between pixel 1 and pixel 2
```
A point block has:
```python
dims = ('n', 'spatial')     # where spatial stands for "spatial coordinates"
coords = (range(0, npoints), ['x', 'y', 'z'])
```
I would have preferred `coords` over `spatial`, but xarray allows for attribute access to coordinates, and having it named the same as the `coords` attribute it would block it off (`data.spatial` now works).

Reply:
Just to clarify: here in `OrientationBlock` I called them `spatial1` and `spatial2` for lack of a better term. Ultimately it was just a trick to automatically see if it should use xy or xyz for the unit vector and similar. I don't think it's an elegant solution, but it's a bit less clunky than what we had. Do you have better ideas?